chia/clvm/spend_sim.py

```diff
@@ -114,15 +124,16 @@ async def create(cls, db_path=None, defaults=DEFAULT_CONSTANTS):
                 self.block_height = store_data.block_height
                 self.block_records = store_data.block_records
                 self.blocks = store_data.blocks
-                self.mempool_manager.peak = self.block_records[-1]
+                # Created a protocol to make BlockRecord and SimBlockRecord interchangeable.
```

Comment:
I don't get this comment. The warning suppression below suggests they aren't interchangeable.